enterprise/security/src/test/resources/ad_test_data.ldif

```diff
@@ -0,0 +1,79 @@
+# LDIF Export for dc=example,dc=com
+# Server: Neo4j Test LDAP Server (10.0.2.15)
+# Search Scope: sub
+# Search Filter: (objectClass=*)
+# Total Entries: 14
+#
+# Generated by phpLDAPadmin (http://phpldapadmin.sourceforge.net) on June 20, 2016 8:43 am
+# Version: 1.2.2
+
+version: 1
+
+# Entry 2: cn=admin,dc=example,dc=com
+dn: cn=admin,dc=example,dc=com
+cn: admin
+description: LDAP administrator
+objectclass: simpleSecurityObject
+objectclass: organizationalRole
+userpassword: {SSHA}szBKVyl4liWB/3FhbbpuYk1bf6dclbyD
+
+# Entry 9: ou=users,dc=example,dc=com
+dn: ou=users,dc=example,dc=com
+objectclass: organizationalUnit
+objectclass: top
+ou: users
+
+# Entry 10: cn=neo4j,ou=users,dc=example,dc=com
+dn: cn=n neo4j,ou=users,dc=example,dc=com
+memberOf: cn=admin,ou=groups,dc=example,dc=com
+objectClass: top
+objectClass: inetOrgPerson
+objectclass: simulatedMicrosoftSecurityPrincipal
+samAccountName: neo4j
+cn: neo4j
```

Comment:
Should this be `cn: n neo4j`?

I actually find the cn `n neo4j` really weird. Why did you take that?